internal/metamorph/processor.go

```diff
@@ -86,6 +86,14 @@ type Processor struct {
 
 	processMinedInterval  time.Duration
 	processMinedBatchSize int
+
+	announcedTransactions     []AnnouncedTransaction
+	announcedTransactionsLock sync.Mutex
+}
+
+type AnnouncedTransaction struct {
+	second uint64
```

Comment:
```suggestion
	timestamp uint64
```

Reply:
It's not actually timestamp, it's simply number of seconds.

Reply:
It's the number of seconds since the Unix epoch. It is used to determine if an object is older than a specific moment in time. Seems like a timestamp to me